akvo/rsr/tests/commands/test_delete_untitled_and_unpublished_projects.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the Akvo RSR module.
+For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+from datetime import datetime, timedelta
+from unittest import mock
+from django.core import management
+from django.utils import timezone
+from django.core.files import File
+from django.core.files.temp import NamedTemporaryFile
+from akvo.rsr.tests.base import BaseTestCase
+from akvo.rsr.models import Project, IatiImport, IatiImportJob, IatiActivityImport, Result, Indicator, IndicatorPeriod, IndicatorPeriodData
+from akvo.rsr.tests.iati_import.xml_files import IATI_V2_STRING
+
+
+class DeleteUntitledAndUnpublishedProjectsCommand(BaseTestCase):
+
+    TWO_DAYS_AGO = timezone.now() - timedelta(days=2)
+    SEVEN_DAYS_AGO = timezone.now() - timedelta(days=7)
+
+    def delete_before_two_days_ago(self):
+        management.call_command('delete_untitled_and_unpublished_projects', self.TWO_DAYS_AGO.strftime('%Y-%m-%d'), '--delete', '--quiet')
+
+    def make_project(self, title='', published=True, created_at=None):
+        if type(created_at) != datetime:
+            return self.create_project(title, published)
+
+        with mock.patch('django.utils.timezone.now') as mocked_now:
+            mocked_now.return_value = created_at
+            return self.create_project(title, published)
```

Comment:
This could probably be merged into `BaseTestCase.create_project`. Looks pretty useful.

Only thing I'd change is `type(created_at) != datetime` --> `isinstance(created_at, datetime)`. Seems more idiomatic.